### PR TITLE
acorn: Add version 0.1.5

### PR DESCRIPTION
**Pull request by sirredbeard**
Adds [Acorn](https://github.com/acorn-io/acorn) to Main.

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by rashil2000 — CHANGES_REQUESTED** [review]
- Since this is a 64bit app, the architecture 64bit block should be used
- Please follow the order of fields given in the Contributing guide

File: bucket/acorn.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "checkver": {
+        "github": "https://github.com/acorn-io/acorn/",
+        "regex": "/releases/tag/v([\\w.-]+)\""
+    },
+    "autoupdate": {
+        "url": "https://github.com/acorn-io/acorn/releases/download/v$version/acorn-v$version-windows-amd64.zip",
+        "hash": {
+            "url": "https://github.com/acorn-io/acorn/releases/download/v$version/checksums.txt",
+            "regex": "$sha256 acorn-v$version-windows-amd64.zip"
+        }
```
Comment (tech189) [inline bucket/acorn.json:17]:
```suggestion
        "hash": {
            "url": "$baseurl/checksums.txt"
        }
```
Simplified autoupdate hash extraction.

**Comment by tech189** [comment]
Whoops sorry I messed something up!

**Review by tech189 — APPROVED** [review]

File: bucket/acorn.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "0.1.5",
+    "description": "A simple application deployment framework for Kubernetes.",
+    "homepage": "https://github.com/acorn-io/acorn",
```
Comment (rashil2000) [inline bucket/acorn.json:4]:
```suggestion
    "homepage": "https://acorn.io",
```

File: bucket/acorn.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/acorn-io/acorn/releases/download/v0.1.5/acorn-v0.1.5-windows-amd64.zip",
+            "hash": "d0484265f8e8becf70ae000a0219894acefe0c6f7ea22a3c313c5ef46ce8e440"
+        }
+    },
+    "bin": "acorn.exe",
+    "checkver": {
+        "github": "https://github.com/acorn-io/acorn/",
+        "regex": "/releases/tag/v([\\w.-]+)\""
```
Comment (rashil2000) [inline bucket/acorn.json:15]:
```suggestion
        "github": "https://github.com/acorn-io/acorn"
```

Can be simplified

File: bucket/acorn.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    "autoupdate": {
+        "url": "https://github.com/acorn-io/acorn/releases/download/v$version/acorn-v$version-windows-amd64.zip",
```
Comment (rashil2000) [inline bucket/acorn.json:18]:
```suggestion
        "architecture": {
            "64bit": {
                "url": "https://github.com/acorn-io/acorn/releases/download/v$version/acorn-v$version-windows-amd64.zip"
            }
        },
```

File: bucket/acorn.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+        "hash": {
+            "url": "$baseurl/checksums.txt"
+        }
+    }
```
Comment (rashil2000) [inline bucket/acorn.json:22]:
```suggestion
    }
}
```

CI is failing due to this missing bracket

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Pull request merged**